setup/vpool.py

```diff
@@ -113,7 +118,25 @@ def add_vpool(vpool_name, vpool_details, api, storagerouter_ip, proxy_amount=2,
             raise RuntimeError(error_msg)
         else:
             VPoolSetup.LOGGER.info('Creation of vPool `{0}` should have succeeded on storagerouter `{1}`'.format(vpool_name, storagerouter_ip))
-            return storagerouter_ip, '/mnt/{0}'.format(vpool_name)
+
+        # Settings volumedriver
+        storagedriver_config = {}
+        if vpool_details.get('storagedriver').get('volume_manager') is not None:
```

Comment:
If storagedriver would be filled in, I'd opt to validate the possible options (see how we validate the call params)
So something like: 
storagedriver_config = vpool_details.get('storagedriver')
if storagedriver_config (not None or empty dict) -> validate this item before proceeding